src/ucp/core/ucp_ep.c

```diff
@@ -1331,7 +1331,8 @@ static void ucp_ep_config_set_rndv_thresh(ucp_worker_t *worker,
                                           ucp_lane_index_t *lanes,
                                           size_t min_rndv_thresh,
                                           size_t max_rndv_thresh,
-                                          ucp_rndv_thresh_t *thresh)
+                                          ucp_rndv_thresh_t *thresh,
```

Comment:
minor: should we add _p suffix for `thresh` and `max_short_to_adjust` arguments?

Reply:
we do not have common rule for that. Is it to mark `out` parameters or relevant for `worker`, `config` and `lanes` as well?